src/panel_gwalker/_tabular_data.py

```diff
@@ -0,0 +1,72 @@
+from typing import TYPE_CHECKING, Any, Union
```

Comment:
This is one of the key changes. Adding support for more general `Tabulator` data than pandas `DataFrame`.